index.html

```diff
@@ -222,5 +223,31 @@ <h2>Get started</h2>
                     <div class="more"><button type="button" class="rectangle animate">Get my CV<img src="images/download-blue.png" alt="download"></button></div>
                     <div class="fillcolor"></div>
                 </section>
+                <div id="projectDetails" class="modal">
+                    <!-- Dectop Modal content -->
+                    <div class="modal-content">
+                        <div class="inner">
+                            <span class="closed">&times;</span>
+                            <h4 id="pdetails_title"></h4>
+                            <ul class="flex" id="languages"></ul>
+                            <div class="slideshow" id="slideshow">
+                            <div class="img-slideshow" id="img-slideshow"></div>
+                            </div>
+                            <div class="sixty">
+                                <p id="text1"></p>
+                                <p id="text2"></p>
+                                <div class="flexfall">
+                                    <button type="button" class="lower big-button center press reverse "><span>See live</span><img class="right" src="images/teal.png" alt="download"></button>
+                                    <button type="button" class="lower big-button center press reversemb "><span>See source</span><img src="images/group.png" alt="download"></button>
+                                </div>
+                                <div class="flex between">
+                                    <button type="button" class="inverse"><span><img src="images/lefty.png" alt="previous"></span>Previous project</button>
+                                    <button type="button" class="inverse end">Next project<span><img src="images/righty.png" alt="previous"></span></button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <!--End of Desctop Modal content -->
+                </div>
```

Comment:
- [ ] Kindly make sure that the projects section and the popup are created dynamically with the information from the projects array. This requires changes in existing code that are already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript.